pkgs/development/interpreters/wasmtime/default.nix

```
@@ -2,24 +2,24 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "wasmtime";
-  version = "0.21.0";
+  version = "0.22.0";
 
   src = fetchFromGitHub {
     owner = "bytecodealliance";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0q7wsnq5zdskxwzsxwm98jfnv2frnwca1dkhwndcn9yyz2gyw57m";
+    sha256 = "1zw8fa8alkxj810gq0kx01id9ankad211ax7x36kqck8ljyiazs1";
     fetchSubmodules = true;
   };
 
-  cargoSha256 = "1r1fm28zaxfbzd17jzaz8ql6ss6y6djgdhpfpkvpbw9l8l06x4lc";
+  cargoSha256 = "0qjaanf62zvk1xk7npg36j53plq5cnls8z549680fhy8rszybzbj";
 
   nativeBuildInputs = [ python cmake clang ];
   buildInputs = [ llvmPackages.libclang ] ++
    lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
```

```suggestion
  buildInputs = [ llvmPackages.libclang ]
    ++ lib.optionals stdenv.isDarwin [ Security ];
```
Please inherit this in top-level and add it as an input.

---

Hmm, any good reason why exposing the Security framework on the top level is beneficial? After all it is only needed  on `Darwin` (and once), not unconditionally. When `grep`ping through the sources, I see all three styles (and  this file was not originally written by me). I have checked in a compromise 1fb2969, which is using `with` and shortens the list, while opening the  frameworks only in the `Darwin` case.

---

> When grep-ping through the sources

That usually include a lot of old packages which should be changed, too.

> Hmm, any good reason why exposing the Security framework on the top level is beneficial?

I don't know the reason right now but darwin like pkgs shouldn't be in inputs. Using with does not solve that.